stylesheets/styles.css

```diff
@@ -70,6 +70,12 @@ body, html {
     border: 0.2em solid rgba(255, 123, 0);
 }
 
+.highlight-listing {
+    background: #ffd699;
+    border: 2px solid red;
+    border-radius: .3em;
+}
```

Comment:
Could we possibly change the border on this from 'px' to 'em' to keep the stylesheet units uniform? 🙂